.doc_gen/metadata/s3-control_metadata.yaml

```diff
@@ -1,7 +1,7 @@
 s3-control_Hello:
   title: Hello &S3Control;
-  title_abbrev: Hello '&S3Control;'
-  synopsis: get started using '&S3Control;'
+  title_abbrev: Hello &S3Control;
+  synopsis: get started using '&S3Control;.
```

Comment:
Lost an apostrophe here, but not both of them

Reply:
Good catch, meant to remove them all.